### PR TITLE
Completed css-canvas-medical exercise -Janice Lee

### PR DESCRIPTION
**Pull request by janiicelee**
Screen Shot 2022-02-01 at 1 03 52 PM
Screen Shot 2022-02-01 at 1 04 23 PM
Screen Shot 2022-02-01 at 1 04 44 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Janice! This page seems too wide as compared to the example, it's especially apparent in the hero image section. Additionally, the button in the navbar is not properly aligned with the links it is next to. Please revise and submit a new screenshot with your revision.

**Comment by janiicelee** [comment]
127 0 0 1_5500_css-canvas-medical_index html (1)

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        <p>Caribou Medical, Towanda PA</p>
+        <p>Bay Tree Family Care, Red Bay AL</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <h2>Ready to learn more?</h2>
+        <p>Schedule a private meeting now</p>
+      </div>
+      <div class="column-one-third"></div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:62]: Empty `column` elements are a no-no. Instead you can control where the `column` elements place themselves in the `row` by adding an additional class with `justify-content` applied to it to position the flexbox's children where you want them. In this case, you would likely use the `space-between` value 🤓

File: css-canvas-medical/index.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        <div class="address">
+          <p>
+            2307 Irving Street, Suite 228<br />
+            San Francisco CA 94122<br />
+            (800) 370- 1416<br/>
+            info@canvasmedical.com
+          </p>
+        </div>
+      </div>
+      <div class="column-fourth"></div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:79]: Same here, no empty columns. For this, you can just make the `column-fourth` before this into a `column-half` to take up that extra space.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by janiicelee** [comment]
127 0 0 1_5500_css-canvas-medical_index html

File: css-canvas-medical/index.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        <p>San Juan Healthcare, Friday Harbor WA</p>
+      </div>
+      <div class="column-half">
+        <p>Ward Medical Clinic, Bonifay FL</p>
+        <p>Caribou Medical, Towanda PA</p>
+        <p>Bay Tree Family Care, Red Bay AL</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-two">
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:58]: `column-two`? This should be named after its measurement.

File: css-canvas-medical/styles.css [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+}
+
+.column-one-third {
+  background: #525351;
+  width: 100%;
+  flex-basis: 35%;
+  justify-content: space-between;
+  padding: 100px 50px;
+  color: white;
+}
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:125]: Remember, the only styling on `column` classes should be either `flex-basis` or `width`. If you need additional styling on a `column` element, you need to make an additional class that you can add **_in addition_** to the `column` class

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, it's looking good, but we don't want to have a column-half and a column-one-half. or a style-half and a style-one-half. Try applying your colors to sections.

```css
.locations {
  color: your grey color;
  background-color: your other grey color;
}
```

**Comment by janiicelee** [comment]
@thebearingedge Hi Tim! So do you mean I have to rename the css ruleset that contains the styles for the columns..? or do I have to remove all of them and make a new class? Regarding locations, I thought it was reasonable to indicate the columns for the location.. Can you give me more advice on this part? Thank you!

**Comment by thebearingedge** [comment]
Hi Janice, please message me on slack

**Comment by janiicelee** [comment]
127 0 0 1_5500_css-canvas-medical_index html (1)

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**